lib/workos/user_management.rb

```diff
@@ -77,7 +77,7 @@ def delete_user(id:)
       def reset_password(token:, new_password:)
         response = execute_request(
           request: post_request(
-            path: '/users/password_reset',
+            path: '/user_management/users/password_reset',
```

Comment:
oh, not everything should have `/user_management/users`. only the REST-style endpoints for User objects... 

the general approach is:
* User Management routes are all "namespaced" under `/user_management`
* within the User Management namespace, we have some REST-style endpoints for User resources. those are `GET /user_management/users/:id` etc
* there are _also_ some endpoints that are part of the User Management product (so using the namespace), but that aren't a REST endpoint on a User resource (like sending a password reset email)

previously, we were treating `/users` as _both_ a product namespace _and_ a resource, and that's what we're moving away from.

check the spreadsheet and/or the API Reference for what these should be 👀

Reply:
Oh cool, I was in automatic mode and rename everywhere 😅 Going to make the fixes here.

Reply:
yeah no worries, i figured! thank youuu 🙏